README.md

```diff
@@ -324,6 +324,17 @@ end
 
 MIT
 
+## Testing
+
+`mix test` runs a variety of tests, most of the use a cluster of
```

Comment:
There's a minor typo: (...) most of them (...).

Reply:
Fixed